tasks/lint.yaml

```diff
@@ -20,4 +27,93 @@ tasks:
     description: Run linting checks on OSCAL
     actions:
       - description: Lint OSCAL files
-        cmd: lula tools lint -f $OSCALFILES
+        cmd: lula tools lint -f "$OSCALFILES"
```

Comment:
iirc this breaky if the '$OSCALFILES' don't exist

Reply:
I can add an if on it - _and_ now that we have this https://github.com/defenseunicorns/uds-common/blob/main/docs/package_integration/oscal-guidelines.md I'll update the default too.

Reply:
added but @brandtkeller thoughts on including this check in Lula directly? - also maybe not bc I think eventually we'd want to force this on you...

Reply:
added a warning for now - we may eventually want to force this on people.

Reply:
(also this is broken until https://github.com/defenseunicorns/uds-common/pull/254 pulls in an update to uds-cli)